main.css

```diff
@@ -0,0 +1,297 @@
+@import url("./define.css");
+html,
+body {
+  margin: 0px;
+  padding: 0px;
```

Comment:
- [optional] giving a `px` value for `0` value is not a good practice. Instead, you should use just `0` like ` margin: 0;`. concider adding this changes.